build.sbt

```diff
@@ -1,39 +1,48 @@
+import scala.util.Properties.isJavaAtLeast
+
 lazy val tokenizer = project
   .in(file("tokenizer"))
   .enablePlugins(Omnidoc, Publish)
 
-lazy val anorm = project
+lazy val core = project
```

Comment:
I think this changes the artifact id of the project (sbt uses a macro that finds the name of the val), we can override it specifically by setting `projectId := "anorm"` or something like that, but it may be easier to leave this as `anorm`, and rename the `anorm` project down lower to `root`.  Also, maybe it's time to restructure the project a little, ie put the core project in a directory called `core`.